packages/ui-extensions/package.json

```diff
@@ -51,6 +66,8 @@
     "./build/cjs/preact.js"
   ],
   "devDependencies": {
+    "@remote-ui/async-subscription": "^2.1.16",
+    "@remote-ui/core": "^2.2.5",
```

Comment:
These were production dependencies before https://github.com/Shopify/ui-extensions/blob/d4020313fd35ad838e2fd06b433487b67e4f12ad/packages/ui-extensions/package.json#L66-L69

Was that because we imported actual code from them?